src/app/view/timestamp-table/timestamp-table.component.ts

```diff
@@ -90,61 +119,28 @@ export class TimestampTableComponent implements OnInit, OnChanges {
             { label: PORT_LEVEL_FILTER.facilityName, value: PORT_LEVEL_FILTER }
           ]
           for (let terminal of terminals) {
-            this.filterTerminals.push({label: terminal.facilitySMDGCode, value: terminal})
+            this.filterTerminals.push({ label: terminal.facilitySMDGCode, value: terminal })
           }
         })
     }
-    this.loadTimestamps(true); // populate PortCallParts at start
-  }
-
-  public isPrimaryReceiver(timestampInfo: TimestampInfo): boolean {
-    return this.hasOverlap(
-      this.globals.config.publisherRoles,
-      timestampInfo.timestampDefinitionTO.publisherPattern.map(x => x.primaryReceiver)
-    )
   }
 
-  filterTimestampsByPortOfCallPart() {
-    this.loadTimestamps(false, this.selectedPortCallPart);
-    this.filterTimestamps();
-  }
-
-  private populatePortCallParts() {
-    this.selectedPortCallPart = null;
-    let uniqueParts = new Set(this.timestampInfos.map(timestamp => timestamp.timestampDefinitionTO?.portCallPart).filter((value) => {
-      return value !== undefined;
-    }));
-
-    this.portCallParts.push({ label: this.translate.instant('general.portCallParts.select'), value: null });
-
-    uniqueParts.forEach(portCallPart => {
-      this.portCallParts.push({ label: portCallPart, value: portCallPart })
-    })
-
-
-  }
-
-  private loadTimestamps(populateFilters?: boolean, portOfCallPart?: string) {
-
+  private loadTimestamps() {
     if (this.transportCallSelected) {
       this.progressing = true;
-      this.timestampMappingService.getPortCallTimestampsByTransportCall(this.transportCallSelected, this.filterTerminal, portOfCallPart).subscribe(timestampInfos => {
+      this.timestampMappingService.getPortCallTimestampsByTransportCall(this.transportCallSelected, this.filterTerminal, this.filterPortCallPart).subscribe(timestampInfos => {
```

Comment:
Backend cannot do the filter on portCallPart (only negotiationCycle) as portCallPart backend filtering will break our "latest in negotiation cycle"-marker